src/wallet/wallet.h

```diff
@@ -1139,17 +1139,17 @@ class CWallet final : public CCryptoKeyStore, public CValidationInterface
     /* Returns true if HD is enabled */
     bool IsHDEnabled() const;
 
-    /* Generates a new HD master key (will not be activated) */
-    CPubKey GenerateNewHDMasterKey();
+    /* Generates a new HD seed (will not be activated) */
+    CPubKey GenerateNewSeed();
 
     /* Derives a new HD master key (will not be activated) */
```

Comment:
Post merge nit: Should say seed here as well instead of "master key"?

Reply:
Fixed in #13297